SourceCode/Homework03/CRTScreen.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <vector>
+#include "CRTVector.h"
+#include "CRTColor.h"
+#include "CRTRay.h"
+
+
+const float PIXEL_LENGTH = 1.0;

```

Comment:
Document why would you need something different than 1

Reply:
I imagined it could be useful if we need to scale the rendered objects without changing the coordinates in the scene